### DISCUSSION TITLE
Cancel Custom Domain Billing

### DISCUSSION THREAD

**Discussion by Ndidi** [supabase/supabase#29428] [Questions] [answered]
I thought I needed a custom domain and have been paying for it for a few months. I now understand my use case better and realise I don't need it.

I've removed the Custom Domain from my project, however I don't see where I can cancel the billing for it. There are no docs that clearly outline how this is to be done. Please help 🙏🏿

**Comment by GaryAustin1** [comment, marked as answer]
Did you check the addon page?
![image](https://github.com/user-attachments/assets/32c9724b-f5c9-41c2-b45b-b2007ca367e2)
